scripts/convolve-mtss-rotamers.py

```diff
@@ -47,11 +47,17 @@
         parser.add_option("--clashDistance", dest="clashDistance", type=float, default=2.2,
                           help="the distance between heavy-atoms of the label and protein within which "
                                "they are assumed to be clashing [%default Angstroms]")
-        parser.add_option("--output", dest="outputFile", default="output.dat",
+        parser.add_option("--outputHistogram", dest="outputFileHistogram", default="output.dat",
```

Comment:
For compatibility, also make `--output` an option (might be as simple as adding it after `"--outputHistogram"`... check). But we don't want to break old code.